go.mod

```diff
@@ -85,6 +85,7 @@ require (
 	github.com/cockroachdb/pebble v0.0.0-20220817183557-09c6e030a677 // indirect
 	github.com/cockroachdb/redact v1.0.8 // indirect
 	github.com/cockroachdb/sentry-go v0.6.1-cockroachdb.2 // indirect
+	github.com/cosmos/cosmos-sdk/tests v0.0.0-20230104083136-11f46a0bae58 // indirect
```

Comment:
what leads to this import?

Reply:
oops, I forgot to remove this.
I just happened to do this while doing some testing locally, removed now.